std_msgs/Header header
MonoMeas[] mono_features
